File: SummaryPro6/app/src/main/java/com/solie/summarypro6/MainActivity.java
package com.solie.summarypro6;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AppHelper.openDatabase(getApplicationContext(),"movie");
        AppHelper.CreateTable("outline");

    }
}

File: Ch_Nyang/app/src/main/java/com/jcorp/ch_nyang/MainActivity.kt
package com.jcorp.ch_nyang

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.jcorp.ch_nyang.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding : ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        onOpenMap()

    }
    internal fun onOpenMap() {
        supportFragmentManager.beginTransaction()
            .replace(R.id.mainContainer, MapFragment())
            .commitNow()
    }
}